project_one/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Fuzail's Hotel</title>
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+    <header>
+        <div class="main">
+            <div class="logo">
+                <img src="logo.png">
+            </div>
+            <ul>
+                <li id="grading-element-one" onclick="myFunctionOne()"><a id="grading-element-five" href="#">Call Us</a></li>
+                <li class="active"><a id="grading-element-five" href="index.html">Home</a></li>
+                <li><a id="grading-element-five" href="">Book A Room</a></li>
+                <li><a id="grading-element-five" href="about.html" id="grading-element-two">About</a></li>
```

Comment:
you declare two `id=` in this `a` tag...kill one and this test will work as expected.